src/test/resources/application-unit.yml

```diff
@@ -0,0 +1,10 @@
+# Config overrides specific to unit tests.
+janitor:
+  primary:
+    # Testing is easier if we are not scheduling changes to the database by default.
+    scheduler-enabled: false
+  pubsub:
+    track-resource:
+      enabled: false
+  stackdriver:
+    enabled: false
```

Comment:
disabling stackdriver is probably good for the application-test default, thought it's kind of harmless. I think the exporters will fail to set up reasonably.

Reply:
Good call – done.